packages/app/cypress/e2e/reporter_header.cy.ts

```diff
@@ -8,23 +8,6 @@ describe('Reporter Header', () => {
   })
 
   context('Specs Shortcut', () => {
-    it('displays the currently running spec as highlighted in the Specs List', () => {
```

Comment:
I will say, validating the correct spec is selected when running is still a valid e2e test case.

Reply:
added it back in 👍